src/quemb/molbe/mbe.py

```diff
@@ -886,139 +892,130 @@ def print_ini(self):
-                    "Pyscf mean field object has to support `with_df`.",
-                )
-                # pyscf.ao2mo uses DF object in an outcore fashion using (ij|P)
-                #   in pyscf temp directory
-                for I in range(self.fobj.n_frag):
-                    eri = self.mf.with_df.ao2mo(self.Fobjs[I].TA, compact=True)
-                    file_eri.create_dataset(self.Fobjs[I].dname, data=eri)
-            elif int_transform == "int-direct-DF":
-                # If ERIs are not saved on memory, compute fragment ERIs integral-direct
-                ensure(bool(self.auxbasis), "`auxbasis` has to be defined.")
-                integral_direct_DF(
-                    self.mf, self.Fobjs, file_eri, auxbasis=self.auxbasis
-                )
-                eri = None
-            elif int_transform == "sparse-DF-cpp":
-                ensure(bool(self.auxbasis), "`auxbasis` has to be defined.")
-                transform_sparse_DF_integral_cpp(
-                    self.mf,
-                    self.Fobjs,
-                    auxbasis=self.auxbasis,
-                    file_eri_handler=file_eri,
-                    MO_coeff_epsilon=self.MO_coeff_epsilon,
-                    AO_coeff_epsilon=self.AO_coeff_epsilon,
-                    n_threads=self.n_threads_integral_transform,
-                )
-                eri = None
-            elif int_transform == "sparse-DF-cpp-gpu":
-                from quemb.molbe.eri_sparse_DF import (  # noqa: PLC0415
-                    transform_sparse_DF_integral_cpp_gpu,
-                )
+        This includes:
 
-                ensure(bool(self.auxbasis), "`auxbasis` has to be defined.")
-                transform_sparse_DF_integral_cpp_gpu(
-                    self.mf,
-                    self.Fobjs,
-                    auxbasis=self.auxbasis,
-                    file_eri_handler=file_eri,
-                    MO_coeff_epsilon=self.MO_coeff_epsilon,
-                    AO_coeff_epsilon=self.AO_coeff_epsilon,
-                    n_threads=self.n_threads_integral_transform,
-                )
-                eri = None
-            elif int_transform == "sparse-DF-nb":
-                ensure(bool(self.auxbasis), "`auxbasis` has to be defined.")
-                transform_sparse_DF_integral_nb(
-                    self.mf,
-                    self.Fobjs,
-                    auxbasis=self.auxbasis,
-                    file_eri_handler=file_eri,
-                    MO_coeff_epsilon=self.MO_coeff_epsilon,
-                    AO_coeff_epsilon=self.AO_coeff_epsilon,
-                    n_threads=self.n_threads_integral_transform,
-                )
-                eri = None
-            elif int_transform == "sparse-DF-nb-gpu":
-                from quemb.molbe.eri_sparse_DF import (  # noqa: PLC0415
-                    transform_sparse_DF_integral_nb_gpu,
-                )
+        - Loading and transforming ERIs (in-core or restored format).
```

Comment:
This is misleading as we don't do basis transformation here. we simply change between different symmetry for the eri